src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/CHIPToolActivity.kt

```diff
@@ -81,11 +81,7 @@ class CHIPToolActivity :
   }
 
   override fun onCommissioningComplete(code: Int) {
-    if (code == 0) {
-      showFragment(OnOffClientFragment.newInstance(), false)
-    } else {
-      showFragment(SelectActionFragment.newInstance(), false)
-    }
+    showFragment(SelectActionFragment.newInstance(), false)
```

Comment:
Can we show a Toast here that says "Commissioning complete: $code"? Otherwise, hard to distinguish errors from successes.

Reply:
+1

Reply:
Ok, done.